Darwin/Examples/StressTestApp/StressTestApp/FoundationInstrumentedHTTPTransport.swift

```diff
@@ -0,0 +1,129 @@
+import Foundation
+import MongoSwift
+@testable import StitchCoreSDK
+
+/**
+ * A basic implementation of the `Transport` protocol using the `URLSession.dataTask` method in the Foundation library.
+ */
+public class FoundationInstrumentedHTTPTransport: FoundationHTTPTransport {
```

Comment:
[minor] I don't like that we're duplicating this. We should probably put the original instrumented transport in some utility module that we can mark as a dependency for this stress test app